src/table.ts

```diff
@@ -119,8 +119,8 @@ export interface JobLoadMetadata {
     field?: string;
     requirePartitionFilter?: boolean;
     type?: string;
-    writeDisposition?: string;
   };
+  writeDisposition?: 'WRITE_TRUNCATE'|'WRITE_APPEND'|'WRITE_EMPTY';
```

Comment:
Thank you for this. Would you mind taking out the specific string values, however? The upstream is the official resource for acceptable values, and we don't want to duplicate from this layer in the event it is changed upstream.

Reply:
No problem. I think it improve typing, but I understand your reasoning for wanting to keep it simple.

Keep in mind, I copied the definitions from `CopyTableMetadata` which also includes specific string values. 😉 
See: https://github.com/googleapis/nodejs-bigquery/blob/master/src/table.ts#L151-L157

I changed the type back to an optional `string` and amended/rebased the commit.

Reply:
Oh, gotcha. I didn't realize it was there, too! Thank you for all of the help!